### PR TITLE
Added new textures to multiblocks

### PR DESCRIPTION
**Pull request by Supernoobv**
# What
Added new Textures to some multiblocks

# Implementation Details
This changes the current textuers for most of the multiblocks from the default in gtceu to unique ones.

### CODE REVIEW

File: src/main/java/supersymmetry/client/renderer/textures/SusyTextures.java [hunk 1]
```diff
@@ -41,6 +41,34 @@ public SusyTextures(){
     public static final OrientedOverlayRenderer FLUID_COMPRESSOR_OVERLAY = new OrientedOverlayRenderer("machines/fluid_compressor");
     public static final OrientedOverlayRenderer FLUID_DECOMPRESSOR_OVERLAY = new OrientedOverlayRenderer("machines/fluid_decompressor");
     public static final OrientedOverlayRenderer ELECTROSTATIC_SEPARATOR_OVERLAY = new OrientedOverlayRenderer("machines/electrostatic_separator");
+    public static final OrientedOverlayRenderer ARC_FURNACE_OVERLAY = new OrientedOverlayRenderer("machines/multiblocks/arc_furnace");
+    public static final OrientedOverlayRenderer CLARIFIER_OVERLAY = new OrientedOverlayRenderer("machines/multiblocks/clarifier");
+    public static final OrientedOverlayRenderer CONDENSER_OVERLAY = new OrientedOverlayRenderer("machines/multiblocks/condenser");
+    public static final OrientedOverlayRenderer COOLING_UNIT_OVERLAY = new OrientedOverlayRenderer("machines/multiblocks/cooling_unit");
+    // not typing allat
```
Comment (planetme) [inline src/main/java/supersymmetry/client/renderer/textures/SusyTextures.java:48]: remove

**Review by planetme — CHANGES_REQUESTED** [review]
I think the arc furnace one is kinda weird, when it is active, the entire rod inside should glow.

File: src/main/java/supersymmetry/client/renderer/textures/SusyTextures.java (hunk 1 above)
Comment (Supernoobv) [reply src/main/java/supersymmetry/client/renderer/textures/SusyTextures.java:45]: Done

**Review by loxoDev — APPROVED** [review]

File: src/main/java/supersymmetry/client/renderer/textures/SusyTextures.java (hunk 1 above)
Comment (planetme) [reply src/main/java/supersymmetry/client/renderer/textures/SusyTextures.java:45]: oh I meant as in remove the comment

**Comment by planetme** [comment]
oh I meant as in remove the comment, not the 3 overlays

you can safely revert 42c5dc2

**Review by planetme — APPROVED** [review]
loooks good to me, but we might want to add some more detailing to the following overlays later:
- arc furnace
- rotary kiln

**Pull request merged**